packet_pool.go

```diff
@@ -56,24 +60,22 @@ func (b *packetAccumulator) add(p *Packet) (ps []*Packet) {
 
 // packetPool represents a queue of packets for each PID in the stream
 type packetPool struct {
-	b map[uint16]*packetAccumulator // Indexed by PID
-	m *sync.Mutex
+	b map[uint32]*packetAccumulator // Indexed by PID
```

Comment:
Can you add the following comment on top of `b`: `// We use map[uint32] instead map[uint16] as go runtime provide optimized hash functions for (u)int32/64 keys`

Reply:
Done